src/routes/group.route.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
```

Comment:
since its a resource, the file name should be in plural , `groups` plural its a better naming convention for rest resources